_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,183 @@
+  the PR description include: Bitcoin Core will now more strongly prefer
+  requesting transactions from *outbound peers*---peers where the
+  connection was opened by the local node; Bitcoin Core will now request
+  an unlimited number of transactions from a peer rather than stopping
+  at 100 (though slow responders won't be preferred); and Bitcoin Core
+  won't keep track of as many pending transaction requests as before,
+  the previous limit being seen as excessive.  The change should also
+  make it easier to test that the transaction request logic is working
+  as expected, with multiple tests for that included in the merged PR.
+  This may help reduce the chance that a flaw can be exploited against
+  users of time-sensitive transactions (such as in LN and many other
+  contract protocols).
+
+- [Bitcoin Core #19077][] wallet: Add sqlite as an alternative wallet database and use it for new descriptor wallets FIXME:dongcarl
+
+- [Bitcoin Core #19770][] RPC: getpeerinfo: Deprecate "whitelisted" field (replaced by "permissions") FIXME:bitschmidty
+
+- [Bitcoin Core #17428][] periodically writes a file with the network
+  addresses of the node's two outbound block-relay-only peers.  The next
+  time the node starts, it reads this file and attempts to reconnect to
+  those same two peers.  This prevents an attacker with the ability to
+  crash a node from being able to trigger a complete change in peers,
+  which would be something they could use as part of an [eclipse
+  attack][topic eclipse attacks] that could potentially be used to trick
+  the node into accepting invalid bitcoins.
+
+- [LND #4688][] adds a new `--duration` parameter that can be used to
+  indicate how long until LND stops making additional attempts to send a
+  payment.  The default remains 60 seconds.
+
+- [Libsecp256k1 #830][] enables GLV endomorphism, which allows verifying
+  signatures with up to 33% fewer operations (see a comment to the PR
+  with actual [performance test results][o'beirne bitcoinperf]).  The PR
+  also removes the slower code for operating without endomorphism as
+  there's no reason to use it now that the patent on the GLV technique
+  has expired (see [Newsletter #117][news117 glv]).  This PR also
+  implements an independent `memcmp` (memory compare) function to avoid
+  the problems with GCC's internal version of that function (see
+  [another section][news117 memcmp] in Newsletter #117).  These changes
+  were subsequently [pulled][Bitcoin Core #20147] into Bitcoin Core.
+
+<!-- FIXME: harding to update topic references Tuesday -->
+{% include references.md %}
+{% include linkers/issues.md issues="19953,17977,19988,19077,19770,17428,4688,830,20147" %}
+[sipa summary]: https://github.com/bitcoin/bitcoin/pull/19953#issuecomment-691815830
+[sipa consensus]: https://github.com/sipa/bitcoin/compare/450d2b2371...865d2c37e2
+[sipa tests]: https://github.com/sipa/bitcoin/compare/206fb180ec...0e2a5e448f426219a6464b9aaadcc715534114e6
```

Comment:
Can these point to bitcoin/bitcoin hashes now that the commits are merged into master?